flocker/provision/_digitalocean.py

```diff
@@ -235,50 +284,37 @@ def provision_digitalocean(node, token,
     # * https://www.digitalocean.com/community/questions/does-libcloud-work-with-digitalocean-s-v2-api # noqa
     # * https://issues.apache.org/jira/browse/JCLOUDS-613
 
-    # Import here, so that this can be added to ``flocker.provision`` without
-    # having to install ``pyocean``.
-    import pyocean
-
-    v2client = pyocean.DigitalOcean(access_token=token)
-    v2droplet = v2client.droplet.get(node._node.id)
-    do_kernel = get_droplet_kernel(v2droplet, DIGITALOCEAN_KERNEL)
-
-    retry_on_error(
-        [pending_event],
-        v2droplet.change_kernel, do_kernel.id)
-
-    run(
-        username='root',
-        address=node.address,
-        commands=task_install_digitalocean_kernel()
-    )
-
-    # Libcloud doesn't support shutting down DO vms.
-    # See https://issues.apache.org/jira/browse/LIBCLOUD-655
-    retry_on_error(
-        [pending_event],
-        v2droplet.shutdown)
-
-    # Libcloud doesn't support powering up DO vms.
-    # See https://issues.apache.org/jira/browse/LIBCLOUD-655
-    # Even after the shutdown, the droplet may not be quite ready to power on,
-    # so also check for that resulting error here.
-    retry_on_error([pending_event, droplet_still_on], v2droplet.power_on)
-
-    # Finally run all the standard Fedora20 installation steps.
-    run(
-        username='root',
-        address=node.address,
-        commands=provision(
-            package_source=package_source,
-            distribution=node.distribution,
-            variants=variants,
-        )
-        # https://clusterhq.atlassian.net/browse/FLOC-1550
-        # This should be part of ._install.configure_cluster
-        + task_open_control_firewall()
-    )
-    return node.address
+    return sequence([
+        # Change the configured kernel
+        Effect(Func(
```

Comment:
I"m using it for expediency, for the moment, since I don't think it is worth defining intents for this now.